compiler/package.go

```diff
@@ -113,6 +113,9 @@ func (pi packageImporter) Import(path string) (*types.Package, error) {
 		}
 		return nil, err
 	}
```

Comment:
The `a, err` are coming from `pi.importContext.Import(path)`.

Via some indirection, that ends up calling:

https://github.com/gopherjs/gopherjs/blob/f8510c9aef19750ec5a1178b33932a7f9defd197/build/build.go#L401-L418

Which is code we control. I think the root problem is somewhere else; when returning `pkg.Archive, nil`, the value of `pkg.Archive` should not be `nil` I would imagine. We should find out why it's possible for it to be.

This fix may work as a temporary solution, but it's better to fix the issue at the root instead (given my assumption that a `nil` value for `pkg.Archive` should never occur under normal conditions).